services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -100,8 +100,6 @@ def check_auth_credentials(cls, values):
             user, pw = values.get("user"), values.get("pw")
             if user is None or pw is None:
                 raise ValueError("Cannot authenticate without credentials user, pw")
-            if not values["ssl"]:
-                raise ValueError("Authentication REQUIRES a secured channel")
```

Comment:
not sure why this goes away?

Reply:
Because it dose not work in development mode with the local registry otherwise. I had to remove it. No other service has this limitation.

Reply:
but does the director-v2 now access the registry??

Reply:
- director-v2 should not access the registry. Right now it should be the director and it will be moved to the catalog ASAP
- in development mode, probably you do not require authentication (``auth=False``) and therefore ``ssl`` is not required

Reply:
Indeed, that's why it was removed.

Reply:
Also, the only reason it needs the registry configuration is because it needs to inject the registry URLs when assembling the docker-compose specs!

Reply:
Isn't there a risk of sending user/password in a non-secure channel as the message that you deleted indicates?

Reply:
> Isn't there a risk of sending user/password in a non-secure channel as the message that you deleted indicates?

It would be the case. But not in this case. The stack's registry is not exposed to the outside world